python/test/regression/test_functional_regressions.py

```diff
@@ -224,3 +224,18 @@ def grid(META):
         BLOCK_SIZE_M=BLOCK_SIZE_M, BLOCK_SIZE_N=BLOCK_SIZE_N, BLOCK_SIZE_K=BLOCK_SIZE_K, type=type,  #
         num_stages=num_stages)
     torch.testing.assert_close(torch_output, triton_output, rtol=1e-2, atol=1e-2)
+
+
+def test_flip(device):
```

Comment:
`test_reverse_range` might be better. Note that we do have a `flip` op

Reply:
Fixed